### ISSUE TITLE
Alter pgaudit extension from pg15 to pg16 not working

### ISSUE THREAD

**Issue by pc-dok** [pgaudit/pgaudit#245]
hi all, 
why pgaudit alter extension will not work anymore from postgres 15 to postgres 16? when i make alter in older versions so from 13 -> 14 and from 14 -> 15 this will work. is that normal behaviour?

regards
franco

**Comment by dwsteele** [comment]
Yes, this normal behavior. You'll need to install pgAudit fresh after an upgrade. The extension does not hold any state so nothing will be lost.
